toxcore/BUILD.bazel

```diff
@@ -988,9 +1004,11 @@ cc_library(
         ":mem",
         ":mono_time",
         ":net_crypto",
+        ":net_profile",
         ":network",
         ":onion_client",
         ":state",
+        ":TCP_server",
```

Comment:
does this need to be here?

Reply:
Yes. tox_private needs a function from TCP_server